rfcs/20230622-quantized-reduction.md

```diff
@@ -0,0 +1,384 @@
+# RFC: StableHLO quantization for reduction ops
+
+Status: Review<br/>
```

Comment:
Per the discussion on OpenXLA Discuss, please change the status to approved and link to the OpenXLA discuss post at: https://groups.google.com/a/openxla.org/g/openxla-discuss/c/iwE9is49SS4

Reply:
I'll do this in a follow up PR actually